module/app/app.go

```diff
@@ -293,18 +298,22 @@ func NewGravityApp(
 
 	bApp.SetParamStore(paramsKeeper.Subspace(baseapp.Paramspace).WithKeyTable(paramskeeper.ConsensusParamsKeyTable()))
 
-	capabilityKeeper := capabilitykeeper.NewKeeper(
+	capabilityKeeper := *capabilitykeeper.NewKeeper(
 		appCodec,
 		keys[capabilitytypes.StoreKey],
 		memKeys[capabilitytypes.MemStoreKey],
 	)
-	app.capabilityKeeper = capabilityKeeper
+	app.capabilityKeeper = &capabilityKeeper
+
+	scopedIBCKeeper := capabilityKeeper.ScopeToModule(ibchost.ModuleName)
+	app.ScopedIBCKeeper = &scopedIBCKeeper
+
+	scopedTransferKeeper := capabilityKeeper.ScopeToModule(ibctransfertypes.ModuleName)
+	app.ScopedTransferKeeper = &scopedTransferKeeper
 
-	scopedIBCKeeper := app.capabilityKeeper.ScopeToModule(ibchost.ModuleName)
-	scopedTransferKeeper := app.capabilityKeeper.ScopeToModule(ibctransfertypes.ModuleName)
 	// Applications that wish to enforce statically created ScopedKeepers should call `Seal` after creating
 	// their scoped modules in `NewApp` with `ScopeToModule`
-	app.capabilityKeeper.Seal()
+	capabilityKeeper.Seal()
```

Comment:
Any use of app.struct has been changed to just struct, because now we're using references everywhere.